app/scrapers/recipe_scraper.rb

```diff
@@ -0,0 +1,37 @@
+class RecipeScraper
+  include Scraper
+
+  def search_results
+    title = query.gsub(/\((\d{2,4})\)/, '').strip
+    edamam_client.search(title)['hits']
+  end
+
+  def scrape_name(result)
+    result['recipe']['label']
+  end
+
+  def scrape_description(result)
+    result['recipe']['ingredientLines'].join("\n")
+  end
+
+  def scrape_image(result)
+    result['recipe']['image']
+  end
+
+  def scrape_date(result)
+  end
+
+  def scrape_tags(result)
+    result['recipe']['healthLabels']
+  end
+
+  def scrape_links(result)
+    [result['recipe']['url']]
+  end
```

Comment:
Links have to be an Array of `Link`s. See other scrapers. Please update the `Scraper` doc so that it mentions that `scrape_links` should return an Array of `Link`s while at it 😄 .